indexing-service/src/main/java/io/druid/indexing/common/actions/SegmentListUsedAction.java

```diff
@@ -53,9 +69,9 @@ public String getDataSource()
   }
 
   @JsonProperty
```

Comment:
@xvrl given that upgrade ordering is... historical, overlord, middle manager, broker, coordinator
can you please tell the scenario where this will fail? (note that it can still deserialize old json with "interval" fine, but it will no longer produce "interval" on serialization.. I have added few serde tests too)